web/app/scripts/src/directives/form.es.js

```diff
@@ -0,0 +1,34 @@
+import { angular } from '../libraries';
+
+import FieldsetParser from '../services/fieldsets-parser';
+
+export const moduleProperties = {
+    moduleName: 'incuna-surveys.form-directive'
+};
+
+const module = angular.module(moduleProperties.moduleName, [
+    FieldsetParser.moduleName
+]);
+
+module.directive('surveysForm', [
+    FieldsetParser.componentName,
+    function (FieldsetParserService) {
+        return {
+            restrict: 'A',
+            scope: {
+                formStructure: '='
+            },
+            template: '<formly-form model="model" fields="fields"></formly-form>',
```

Comment:
For some strange reason this errors when used as an attribute. I filed a bug https://github.com/formly-js/angular-formly/issues/708